lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
I still don't see the value here. What this module [does](https://github.com/sindresorhus/serialize-error/blob/master/index.js) is a weaker variant of what our existing JSONSerializer module does (since it doesn't handle regexes, not that we need them for errors but oh well).

Can we remove this dependency as well and just use the JSONSerializer?